cernopendata/templates/cernopendata_records_ui/docs/detail.html

```diff
@@ -1,64 +1,62 @@
 {%- extends "invenio_records_ui/detail.html" %}
 
 {%- block page_body %}
-<div class="container-fluid background">
-    <div class="container detail-page" ng-app="recordApp">
-        <div class="row">
-            <div class="{% if record.related %} col-md-10 {% else %} col-md {% endif %}">
-                <div class="card card-style">
-                    <div class="card-body">
-                        <h1 class="d-inline"> {{ record.title }} </h1>
-                        <p>
-                        {% if record.get('created') %} <small>{{ record.get('created') }} by </small> {% endif %}
-                        <small>{{ record.author}} </small>
-                        </p>
-                        <p class="badges-box">
-                        {% if record.type %} <a class="badge badge-type" href="/search?type={{ record.type.primary }}">{{ record.type.primary }}</a>
-                        {% if record.type.secondary %}
-                        {% for type in record.type.secondary %}
-                        <a class="badge badge-subtype" href="/search?type={{ record.type.primary }}&subtype={{ type }}">{{ type }}</a>
-                        {% endfor %}
-                        {% endif %}
+<div class="ui {% if record.related %}two{% else %}one{% endif %} centered column stackable grid container detail-page">
+    <div class="row">
+        <div class="{% if record.related %}thirteen wide column{% else %}sixteen wide column{% endif %}">
+            <div class="ui segment">
+                <div class="card-body">
+                    <h1> {{ record.title }} </h1>
+                    <p>
+                    {% if record.get('created') %} <small>{{ record.get('created') }} by </small> {% endif %}
+                    <small>{{ record.author}} </small>
+                    </p>
+                    <p class="badges-box">
+                    {% if record.type %} <a class="badge badge-type" href="/search?type={{ record.type.primary }}">{{ record.type.primary }}</a>
+                    {% if record.type.secondary %}
+                    {% for type in record.type.secondary %}
+                    <a class="badge badge-subtype" href="/search?type={{ record.type.primary }}&subtype={{ type }}">{{ type }}</a>
```

Comment:
Moderate: Badges don't exist on Semantic UI, we need to use [labels](https://semantic-ui.com/elements/label.html), I've already migrated some of them on my frontpage PR, you can check it [here](https://github.com/cernopendata/opendata.cern.ch/pull/3020/files#diff-73bad381fe3eefd15b2224cdc6149a0e6f3d5b6994fbb37d47ee04bffecdb03dR255)

PS. Beware the comment by @audrium about the hover effect, the might not be fully ready.